### ISSUE TITLE
scaler

### ISSUE THREAD

**Issue by Yunhao696** [Haochen-Wang409/HPM#10]
assert len(optimizer_state["found_inf_per_device"]) > 0, "No inf checks were recorded for this optimizer."
作者大大您好，我在三维医学图像上用您的方法，scaler报错这个，去掉scaler后loss完全不变，参数没有更新，请问可能是什么问题

**Comment by Haochen-Wang409** [comment]
您具体是怎么“去掉”scaler的？修改了哪些代码？

**Comment by Yunhao696** [comment]
![image](https://github.com/Haochen-Wang409/HPM/assets/94282580/6a1566c0-2580-4a06-b494-1a203f6184b5)
我将这部分换成了传统的 loss.backward() optimizer.step()

**Comment by Yunhao696** [comment]
然后loss_learn保持不变，可能是参数完全没有任何更新

**Comment by Haochen-Wang409** [comment]
在其他的数据集，如ImageNet上，可以正常运行吗？

**Comment by Yunhao696** [comment]
我将试一下其他的三维数据集，谢谢您

**Comment by Haochen-Wang409** [comment]
建议您在ImageNet数据集先跑通，因为我们并没有在其他数据集上进行预训练。
